drivers/ms8607/ms8607.c

```diff
@@ -0,0 +1,212 @@
+	dev->params = *params;
+	/* acquire bus */
+	if (i2c_acquire(dev->params.i2c_dev) != 0) {
+		DEBUG("[ms8607] error: unable to acquire bus\n");
+		return MS8607_ERR_BUS;
+	}
+	/* reset PT */
+	if (i2c_write_byte(dev->params.i2c_dev, dev->params.i2c_addr_pt,
+			MS8607_CMD_PT_RST, 0) != 0) {
+		DEBUG("[ms8607] error: unable to reset PT\n");
+		goto err;
+	}
+	/* read the compensation data from the sensor's PROM */
+	if (_read_calibration_data(dev) != MS8607_OK) {
+		DEBUG("[ms8607] error: could not read calibration data\n");
+		goto err;
+	}
+	/* reset H */
+	if (i2c_write_byte(dev->params.i2c_dev, dev->params.i2c_addr_h,
+			MS8607_CMD_H_RST, 0) != 0) {
+		DEBUG("[ms8607] error: unable to reset H\n");
+		goto err;
+	}
+	/* init user register */
+	if ((i2c_write_byte(dev->params.i2c_dev, dev->params.i2c_addr_h,
+					MS8607_CMD_H_UR_RD, 0) != 0)
+			|| (i2c_read_byte(dev->params.i2c_dev, dev->params.i2c_addr_h, &reg, 0)
+				!= 0)) {
+		DEBUG("[ms8607] error: unable to read user register\n");
+		goto err;
+	}
+	reg &= ~MS8607_CMD_H_UR_OSR_MSK;
+	reg |= dev->params.osr_h;
+	if (i2c_write_reg(dev->params.i2c_dev, dev->params.i2c_addr_h,
+					MS8607_CMD_H_UR_WR, reg, 0) != 0) {
+		DEBUG("[ms8607] error: unable to write user register\n");
+		goto err;
+	}
+	i2c_release(dev->params.i2c_dev);
+	return MS8607_OK;
+err:
+	i2c_release(dev->params.i2c_dev);
+	DEBUG("[ms8607] init: bus error while initializing device\n");
+	return MS8607_ERR_BUS;
+}
+
+static int _read_adc_pt(ms8607_t *dev, uint32_t *uncompensated, uint8_t cmd)
+{
+	uint8_t buf[3];
+
+	/* acquire bus */
+	if (i2c_acquire(dev->params.i2c_dev) != 0) {
+		DEBUG("[ms8607] error: unable to acquire bus\n");
+		goto err;
+	}
+	/* start conversion */
+	if (i2c_write_byte(dev->params.i2c_dev, dev->params.i2c_addr_pt, cmd, 0)
+			!= 0) {
+		DEBUG("[ms8607] error: unable to send conversion command\n");
+		goto err;
+	}
+	xtimer_usleep(20000);
+	/* read adc */
+	if (i2c_write_byte(dev->params.i2c_dev, dev->params.i2c_addr_pt,
+			MS8607_CMD_PT_ADC_ADC_RD, 0) != 0) {
+		DEBUG("[ms8607] error: unable to send adc read command\n");
+		goto err;
+	}
+	if (i2c_read_bytes(dev->params.i2c_dev, dev->params.i2c_addr_pt,
+				buf, 3, 0) != 0) {
+		DEBUG("[ms8607] error: unable to receive data\n");
+		goto err;
+	}
+	*uncompensated = buf[0] << 16 | buf[1] << 8 | buf[2];
+	i2c_release(dev->params.i2c_dev);
+	return MS8607_OK;
+err:
+	i2c_release(dev->params.i2c_dev);
+	DEBUG("[ms8607] read_adc_pt: bus error\n");
+	return MS8607_ERR_BUS;
+}
+
+static int _read_rh(ms8607_t *dev, uint16_t *d3)
+{
+	uint8_t buf[3];
+	int status;
+
+	/* acquire bus */
+	if (i2c_acquire(dev->params.i2c_dev) != 0) {
+		DEBUG("[ms8607] error: unable to acquire bus\n");
+		goto err;
+	}
+	/* start measure */
+	if (i2c_write_byte(dev->params.i2c_dev, dev->params.i2c_addr_h,
+				MS8607_CMD_H_MEAS_NHOLD, 0) != 0) {
+		DEBUG("[ms8607] error: unable to send measure command\n");
+		goto err;
+	}
+	do {
+		xtimer_usleep(15000);
```

Comment:
pense à utiliser la macro US_PER_MS
```suggestion
		xtimer_usleep(15 * US_PER_MS);
```